<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QWidget {
    background-color: #1e1e1e;
    color: #f0f0f0;
}
</string>
   </property>
   <widget class="QTabWidget" name="tab_uart">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>711</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QTabBar::tab {
    background: #2d2d2d;
    color: #cccccc;
    padding: 4px 7px;
    border: 1px solid #3c3c3c;
    border-bottom: none;
    border-top-left-radius: 1px;
    border-top-right-radius: 1px;
    margin-right: 2px;
}

QTabBar::tab:selected {
    background: #007acc; 
    color: #ffffff;
    font-weight: bold;
    border: 1px solid #007acc;
}

QTabWidget::pane {
    border: 1px solid #3c3c3c;
    top: -1px; 
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_tx">
     <attribute name="title">
      <string>TX</string>
     </attribute>
     <widget class="QTextBrowser" name="display_tx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_rx">
     <attribute name="title">
      <string>RX</string>
     </attribute>
     <widget class="QTextBrowser" name="display_rx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_tx_and_rx">
     <attribute name="title">
      <string>TX and RX</string>
     </attribute>
     <widget class="QTextBrowser" name="display_tx_and_rx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>661</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
	border-image: url(:/icons/play-button.svg);
    background-color: transparent;
    border: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>38</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/stop-button.svg);
    background-color: transparent;
    border: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_baudrate_tx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cmb_baudrate_rx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb_com_port_tx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COM PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_com_port_rx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>COM PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_baudrate_tx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BAUDRATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_baudrate_tx_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BAUDRATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_baudrate_tx_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RX SETUP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_setup_tx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TX SETUP</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_com_port_rx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_com_port_tx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_use_rx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enable RX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk_use_tx">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enable TX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(:/icons/download_2.svg);
    background-color: transparent;
    border: none;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="led_runnig">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {
    background-color: #00ff00;
    border-radius: 6px;
    width: 12px;
    height: 12px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="led_stopped">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {
    background-color: #fedb00;
    border-radius: 6px;
    width: 12px;
    height: 12px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="led_error">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {
    background-color: #fe0000;
    border-radius: 6px;
    width: 12px;
    height: 12px;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_runnig">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f0f0f0;
font-family: 'Segoe UI', sans-serif;
font-size: 12pt
</string>
    </property>
    <property name="text">
     <string>Runnig</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_stopped">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f0f0f0;
font-family: 'Segoe UI', sans-serif;
font-size: 12pt
</string>
    </property>
    <property name="text">
     <string>Stopped</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_stopped_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f0f0f0;
font-family: 'Segoe UI', sans-serif;
font-size: 12pt
</string>
    </property>
    <property name="text">
     <string>COM Port Error</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resourcer.qrc"/>
 </resources>
 <connections/>
</ui>
